drivers/power_domain/Kconfig

```diff
@@ -18,6 +18,7 @@ config POWER_DOMAIN_GPIO
 	depends on DT_HAS_POWER_DOMAIN_GPIO_ENABLED
 	depends on GPIO
 	depends on TIMEOUT_64BIT
+	depends on PM_DEVICE
```

Comment:
Is this really being used with CONFIG_PM_DEVICE alone? (PM_DEVICE is quite broken)

Reply:
I'm not sure tbh. I'm was just checking different config options for a driver I'm currently writing and then had compilation errors with this combination.

Even if this combination is unlikely, shouldn't the dependencies be correct? But take it with a grain of salt since I'm not very deep in the PM topic.